### DISCUSSION TITLE
How to get code snippets in different languages side-by-side?

### DISCUSSION THREAD

**Discussion by Robinlovelace** [quarto-dev/quarto-cli#6640] [Q&A] [answered]
### Description

As illustrated below, it can be handy to have code chunks side-by-side:

![image](https://github.com/quarto-dev/quarto-cli/assets/1825120/a553cbf6-c153-4fd5-a8d6-a5b895afb19d)

Source: https://duckdb.org/docs/archive/0.8.1/installation/

How to do that? Currently this is what we have: https://github.com/Robinlovelace/opengeohub2023/blob/main/side-by-side.qmd

Relevant issue: https://github.com/rstudio/rmarkdown/issues/2053

Assuming `.tabset` works (not tested), is there a way to make the value persist, e.g. so that if you select R in the first chunk you get R in all chunks?

**Comment by Robinlovelace** [comment, marked as answer]
OK think I've answered my own question: https://quarto.org/docs/output-formats/html-basics#tabsets

**Reply by mcanouil** [reply]
Indeed, that would be it.
